_includes/banner.html

```diff
@@ -18,4 +19,7 @@ <h1 class="text-lg-center">Giving you a voice in the #OpenSource Tech community
             <p><b>WeRockTech</b> is a youtube channel which so far features discussions & conversations with people in tech who have either contributed by way of their achievements or are new to the community and starting explore how they can contribute & get involved, hopefully this will encourage more to join & do similar</p>
         </div>
     </div>
+
+    <center><p><b>WeRockTech</b> is a youtube channel which so far features discussions & conversations with people in tech who have either contributed by way of their achievements or are new to the community and starting explore how they can contribute & get involved, hopefully this will encourage more to join & do similar</p></center>
```

Comment:
Sounds good 👍 .

Make the changes in your `experiment` branch and this **Pull Request**  will automatically be updated with your latest changes